.gitmodules

```diff
@@ -9,6 +9,7 @@
 	url = https://github.com/FreeRTOS/coreHTTP.git
 [submodule "libraries/standard/coreJSON"]
 	path = libraries/standard/coreJSON
+	branch = main
```

Comment:
Not necessary with the latest, I will put back the master branch as non-default in coreJSON so that older commits won't fail.